delta/extensions/sources/sentinel1.py

```diff
@@ -115,8 +113,11 @@ def unpack_s1_to_folder(zip_path, unpack_folder):
             # - The SNAP tool *must* write to a .tif extension, so we have to
             #   rename the file if we want something else.
             temp_out_path = merged_path.replace('.vrt', '.tif')
-            cmd = (SNAP_SCRIPT_PATH + ' ' + SNAP_GRAPH_PATH + ' '
-                   + unpack_folder + ' ' + temp_out_path)
+            cmd = (SNAP_SCRIPT_PATH + ' ' + unpack_folder + ' ' + temp_out_path)
+            print(cmd)
+            os.system(cmd)
+            dimap_path = temp_out_path + '.dim'
+            cmd = 'pconvert -s 0,0 -f GeoTIFF-BigTiff -o ' + os.path.dirname(temp_out_path) +' '+ dimap_path
```

Comment:
Is it documented somewhere how to install snap? Could you check if the programs are installed and given an error with a link if they aren't?